inject/src/main/java/io/micronaut/inject/writer/AbstractBeanDefinitionBuilder.java

```diff
@@ -118,17 +119,21 @@ public abstract class AbstractBeanDefinitionBuilder implements BeanElementBuilde
 
     /**
      * Default constructor.
-     * @param originatingElement The originating element
-     * @param beanType The bean type
-     * @param metadataBuilder the metadata builder
-     * @param visitorContext the visitor context
+     *
+     * @param originatingElement               The originating element
+     * @param beanType                         The bean type
+     * @param metadataBuilder                  the metadata builder
+     * @param visitorContext                   the visitor context
+     * @param elementAnnotationMetadataFactory The element annotation metadata factory
      */
     protected AbstractBeanDefinitionBuilder(
```

Comment:
We should rename this type to `AbstractBeanElementBuilder` otherwise we have two classes with the same name